types/chrome/index.d.ts

```diff
@@ -3182,7 +3195,7 @@ declare namespace chrome.events {
         /** Optional. Matches if the URL (without fragment identifier) ends with a specified string. Port numbers are stripped from the URL if they match the default port number.  */
         urlSuffix?: string | undefined;
         /** Optional. Matches if the port of the URL is contained in any of the specified port lists. For example [80, 443, [1000, 1200]] matches all requests on port 80, 443 and in the range 1000-1200.  */
-        ports?: any[] | undefined;
+        ports?: (number | number[])[];
```

Comment:
https://developer.chrome.com/docs/extensions/reference/events/#type-UrlFilter

Reply:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        ports?: (number | number[])[] | undefined;
```